cli/cmd/install_test.go

```diff
@@ -172,6 +181,7 @@ func TestRender(t *testing.T) {
 		{haValues, "install_ha_output.golden"},
 		{haWithOverridesValues, "install_ha_with_overrides_output.golden"},
 		{noInitContainerValues, "install_no_init_container.golden"},
+		{withProxyIgnoresValues, "install_proxy_ignores.golden"},
 	}
```

Comment:
Maybe its worth adding a test that fails and outputs the reason, just to ensure the validation logic you wrote will remain wired correctly into the install cmd

Reply:
Added the new `TestValidateAndBuild_Errors` method to test fail cases.